migrations/scripts/v7_layer_migration.py

```diff
@@ -211,7 +212,9 @@ def migrate_crawlers(layer: str, glue_client, resource_client):
 
 
 def format_tags_acceptably_for_crawler_creation(tags: List[dict]) -> dict:
-    return {tag["Key"]: tag["Value"] for tag in tags}
+    new_default_tags = {"db_name": GLUE_CATALOGUE_DB_NAME}
+    existing_formatted_tags = {tag["Key"]: tag["Value"] for tag in tags}
```

Comment:
Please can you move these changes to where the rest of the tags are added - line 259